apps/src/applab/designElements/ZOrderRow.jsx

```diff
@@ -19,13 +19,23 @@ var ZOrderRow = React.createClass({
 
   render: function() {
     var element = this.props.element;
-    var index = Array.prototype.indexOf.call(element.parentNode.children, element);
+
+    // Element will be wrapped in a resizable div
+    var outerElement = element.parentNode;
+    var index = Array.prototype.indexOf.call(outerElement.parentNode.children, outerElement);
     var isBackMost = index === 0;
-    var isFrontMost = index + 1 === element.parentNode.children.length;
+    var isFrontMost = index + 1 === outerElement.parentNode.children.length;
 
     var squareButton = {
       width: 42,
-      height: 42
+      height: 42,
+      backgroundColor: '#0094ca' // $cyan
```

Comment:
I know we've been doing styles inline for React elements, but I wonder if this is a case where it's better to add a class that has its color set in the stylesheet?  Otherwise if these colors ever change we'll have to update here as well.